src/host_context.rs

```diff
@@ -178,13 +178,13 @@ impl HostContext {
             ScObject::ScoU64(u) => self.add_host_object(*u),
             ScObject::ScoI64(i) => self.add_host_object(*i),
             ScObject::ScoString(s) => {
-                let ss = match String::from_utf8(s.clone()) {
+                let ss = match String::from_utf8(s.clone().into()) {
                     Ok(ss) => ss,
                     Err(_) => return Err(()),
                 };
                 self.add_host_object(ss)
             }
-            ScObject::ScoBinary(b) => self.add_host_object(b.clone()),
+            ScObject::ScoBinary(b) => self.add_host_object::<Vec<u8>>(b.clone().into()),
```

Comment:
@graydon Is there a trait I can implement, or a fn that is convention you'd recommend for making these types of calls (`.clone().into()`) more convenient, so we do not need to always specify the destination host object type?